stwfsapy/tests/predictor_test.py

```diff
@@ -364,6 +369,22 @@ def mock_expander(**kwargs):
         assert call(label_text) in stub.mock_calls
 
 
+def test_mark_doc_end_empty():
+    predictor = p.StwfsapyPredictor(None, None, None, None)
+    matches = [('a', 'aa', [0], 0), ('b', 'bbb', [2, 5], 0)]
+    res = predictor._mark_last_concept_in_doc(matches)
+    assert matches == [
+        ('a', 'aa', [0], 0),
+        ('b', 'bbb', [2, 5], 1)]
+
+
+def test_mark_doc():
```

Comment:
```suggestion
def test_mark_doc_end():
    predictor = p.StwfsapyPredictor(None, None, None, None)
    matches = [('a', 'aa', [0], 0), ('b', 'bbb', [2, 5], 0)]
    res = predictor._mark_last_concept_in_doc(matches)
    assert matches == [
        ('a', 'aa', [0], 0),
        ('b', 'bbb', [2, 5], 1)]


def test_mark_doc_end_empty():
```
I think the method names should be swapped?